controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -67,6 +67,7 @@ type DSCInitializationReconciler struct { //nolint:golint,revive // Readability
 	Log                   logr.Logger
 	Recorder              record.EventRecorder
 	ApplicationsNamespace string
+	Audiences             []string
```

Comment:
I am wondering if there's some other way of passing this config down the chain. Somewhat does not feel like this should belong here, it's unrelated to the reconcile at this level. Let's think about possible ways and sync tomorrow.

Reply:
moved it! here
c99d6d7